psychopy/visual/shape.py

```diff
@@ -484,11 +484,11 @@ def _initVertices(self, newVertices):
                                    tesselate.default_winding_rule)
 
         if not self.closeShape or tessVertices == []:
-            # probably got a line if tesselate returned []
+            # probably got a line if tessellate returned []
             initVertices = newVertices
             self.closeShape = False
         elif len(tessVertices) % 3:
-            raise tesselate.TesselateError("Could not properly tesselate")
+            raise tessellate.TesselateError("Could not properly tessellate")
```

Comment:
This one is code (not a comment) so changing it here will break the code

Reply:
yea.. thanks. will change that.